data/transactions/logic/README.md

```diff
@@ -436,58 +540,67 @@ of a true top-level transaction, programatically.  However, they are
 different in significant ways.  The most important differences are
 that they are not signed, duplicates are not rejected, and they do not
 appear in the block in the usual away. Instead, their effects are
-noted in metadata associated with the associated top-level application
+noted in metadata associated with their top-level application
 call transaction.  An inner transaction's `Sender` must be the
 SHA512_256 hash of the application ID (prefixed by "appID"), or an
 account that has been rekeyed to that hash.
 
-Currently, inner transactions may perform `pay`, `axfer`, `acfg`, and
+In v5, inner transactions may perform `pay`, `axfer`, `acfg`, and
 `afrz` effects.  After executing an inner transaction with
 `itxn_submit`, the effects of the transaction are visible begining
 with the next instruction with, for example, `balance` and
-`min_balance` checks.
+`min_balance` checks. In v6, inner transactions may also perform
+`keyreg` and `appl` effects.
 
-Of the transaction Header fields, only a few fields may be set:
-`Type`/`TypeEnum`, `Sender`, and `Fee`. For the specific fields of
-each transaction types, any field, except `RekeyTo` may be set.  This
-allows, for example, clawback transactions, asset opt-ins, and asset
-creates in addtion to the more common uses of `axfer` and `acfg`.  All
-fields default to the zero value, except those described under
-`itxn_begin`.
+In v5, only a few of the Header fields may be set: `Type`/`TypeEnum`,
```

Comment:
consider rephrasing:
`In v5, only a few of the Header fields may be set` -> `In v5, only a subset of the transaction's header fields could have been specified :`